models/public/efficientnet-b7_auto_aug/efficientnet-b7_auto_aug.md

```diff
@@ -5,8 +5,8 @@
 The `efficientnet-b7_auto_aug` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946)
 models designed to perform image classification, trained with the
 [AutoAugmentation preprocessing](https://arxiv.org/abs/1805.09501).
-This model was pretrained in TensorFlow\*.
-All the EfficientNet models have been pretrained on the ImageNet\* image database.
+This model was pre-trained in TensorFlow\*.
+All the EfficientNet models have been pre-trained on the ImageNet\* image database.
```

Comment:
in other files ImageNet named without asterix

Reply:
```suggestion
All the EfficientNet models have been pre-trained on the ImageNet image database.
```

Reply:
Updated